MET Landing page

ElenaGrasovskaya opened the pull request:
https://ElenaGrasovskaya.github.io/layout_landing-page/

sintax1s commented on src/styles/components/_contacts.scss:
```diff
@@ -0,0 +1,94 @@
+      }
+    }
+
+    &-button,
+    &-button:active,
+    &-button:hover {
+      height: 56px;
+      background-color: var(--main-accent);
+      color: var(--main-white);
+      border: none;
+      text-align: center;
+      font-family: inherit;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px; /* 168.75% */
```
your button without any hover and even without cursor pointer, we can't know that the button was pressed

ElenaGrasovskaya replied:
Fixed that

sintax1s commented on src/styles/components/_footer.scss:
```diff
@@ -0,0 +1,12 @@
+.footer {
+  height: 180px;
```
at desktop version your footer too small,must be not less 359px,also footer must be sticky.
![image](https://github.com/mate-academy/layout_landing-page/assets/143347271/49572698-1ff0-4b90-bbc2-5d7e145b57ff)

ElenaGrasovskaya replied:
Improved that as well

sintax1s commented on src/styles/components/_icon.scss:
```diff
@@ -0,0 +1,18 @@
+  height: 24px;
+  width: 24px;
+
+  &--phone {
+    background-image: url(../images/icons/icon-phone.svg);
+    &:hover + .header__phones {
+      opacity: 1;
+    }
+  }
```
At small resolutions your hover works like that,fix it
![image](https://github.com/mate-academy/layout_landing-page/assets/143347271/a3b7dba0-9731-4147-a8f6-8933845e8a21)

ElenaGrasovskaya replied:
It should not appear on the phone resolution, fixed that

sintax1s commented on src/styles/components/_menu.scss:
add for menu some animation

ElenaGrasovskaya replied:
Added

sintax1s commented on src/styles/layout/_page.scss:
```diff
@@ -0,0 +1,34 @@
+$tablet-size: 600px;
+$wide-screen-size: 1000px;
```
better create separate file for variables

ElenaGrasovskaya replied:
Did that

sintax1s commented on src/styles/layout/_page.scss:
```diff
@@ -0,0 +1,34 @@
+$tablet-size: 600px;
+$wide-screen-size: 1000px;
+
+.page {
+  margin: 0;
+  --main-white: #fff;
+  --main-dark: #161616;
+  --main-accent: #d12d35;
+  --main-text: #3e3e3e;
+  --secondary-accent: #ae1c23;
+  --form-background: #f8f8fa;
+  --form-label-grey: #7e7e83;
+
+  font-family: Cinzel, Georgia, Times, "Times New Roman", serif;
+  font-size: 16px;
+  color: var( --main-dark);
+  line-height: 27px;
+}
```
add for your page scroll-behavior: smooth ; for smooth scroll to elements

ElenaGrasovskaya replied:
Did that as well

sintax1s reviewed (changes requested):
Fix my comments,and something more.Add hovers for buttons,also add them transition.In my opinion you need to create container for your content,which will limit the content to 1260px.Good luck.

ElenaGrasovskaya commented:
Thank you for a thoughtful review... the site is probably not perfect still, but I hope it's better now.

superpooperxxx reviewed (changes requested):
Fantastic job🔥

1. Check if all of your phone number are links that start a call
<img width="281" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/96ed97c3-a16c-4902-acad-29b200a6a6e6">

2. Let's add some hover state to nav links
<img width="145" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/260f008e-be69-4520-84af-6a3da3a633a2">

3. Footer should reveal when reaching the end of the page
<img width="380" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/b1614a9a-8d9e-4a8a-a49e-faf4dc6465fd">

4. Let's disable horizontal expanding of textarea
<img width="296" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/0db7e614-e69a-4e77-b753-196f1f7d904c">

ElenaGrasovskaya commented:
Many thanks for the review!
1. Changed phone links from tel: to callto: ... somehow tel: didn't work.
2.  Added lines as hover effect using pseudo-elements
3.  Improved the footer design with a fixed background effect
4. Disabled scaling for text-area

superpooperxxx reviewed (approved):
Good job🔥 I found new bugs😢

1. 
<img width="244" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/b0187328-14ce-4b79-9560-d509ad4ab167">

2. Try hovering here (red rectangles) when phone numbers are hidden.
